### ISSUE TITLE
Formula not supported: TEXTJOIN()

### ISSUE THREAD

**Issue by James-Whitfield** [ClosedXML/ClosedXML#1010]
**Do you want to request a *feature* or report a *bug*?**
- [] Bug
- [X] Feature

**Version of ClosedXML**

0.93.1

**What is the current behavior?**

TEXTJOIN() formula function not supported

**What is the expected behavior or new feature?**

https://support.office.com/en-us/article/textjoin-function-357b449a-ec91-49d0-80c3-0e8fc845691c

**Did this work in previous versions of our tool?  Which versions?**

No

## Reproducibility

**Code to reproduce problem:**
```c#
using ClosedXML.Excel;
using System;

namespace ConsoleApp1
{
    class Program
    {
        static void Main(string[] args)
        {
            var wb = new XLWorkbook();
            var ws = wb.AddWorksheet("Sheet1");
            ws.Cell("A1").FormulaA1 = "=TEXTJOIN(\",\",TRUE,0,1,2,3)";
            // TEXTJOIN() Formula not supported 
            var x = ws.Cell("A1").GetString();
            Console.WriteLine(x);
        }
    }
}

```
- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

**Comment by igitur** [comment]
Thanks. Many formulas are not supported yet. We rely on community contributions to extend ClosedXML in this regard. So have a go at it.

**Comment by James-Whitfield** [comment]
Thank you for the quick reply. I will look into adding support for this.

**Comment by igitur** [comment]
Closed by #1015
